index.html

```diff
@@ -2893,6 +2893,24 @@ <h1>DID Methods</h1>
               <a href="https://github.com/51nodes/schema-registry-did-method/blob/master/README.md">Schema Registry DID Method</a>
           </td>
       </tr>
+
+        <tr>
+          <td>
+            did:key:
+          </td>
+          <td>
+            PROVISIONAL
+          </td>
+          <td>
+            Ledger independent DID method based on public/private key pairs
+          </td>
+          <td>
+            Rick Astley
```

Comment:
@msporny do you have a suggestion to make about this line as well?

Reply:
He was the original inspiration for did:key, so in order to pay homage to him, it would be appropriate to leave it as is.

```suggestion
            Rick Astley (thank you for your inspiration), Manu Sporny, Dmitri Zagidulin, Dave Longley, Orie Steele
```